recipes/pygridgen/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+    name: pygridgen
+    version: 0.1.1
+
+source:
+    git_url: https://github.com/phobson/pygridgen.git
+    git_tag: 0.1.1
+
+build:
+    number: 0
+    skip: True  # [win]
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - numpy
+    run:
+        - python
+        - pandas
+        - numpy
+        - matplotlib
+        - pyproj
+        - gridutils
```

Comment:
Oops :flushed: I wanted to leave only the last on in the stack (`gridgen`) but ended but deleting the wrong one.